cmd/kubelet/app/server.go

```diff
@@ -1159,7 +1160,7 @@ func RunDockershim(f *options.KubeletFlags, c *kubeletconfiginternal.KubeletConf
 		HairpinMode:        kubeletconfiginternal.HairpinMode(c.HairpinMode),
 		NonMasqueradeCIDR:  f.NonMasqueradeCIDR,
 		PluginName:         r.NetworkPluginName,
-		PluginConfDir:      r.CNIConfDir,
+		PluginConfDirs:     cni.SplitDirs(r.CNIConfDir),
```

Comment:
Any reason this split isn't done in the same place as the bindir split dirs, eg in docker_service.go?

Reply:
@dcbw That was done initially https://github.com/kubernetes/kubernetes/pull/69977#discussion_r226640879 I'm fine either way.